RoadMap_VK/RoadMap_VK/Network/Session.swift

```diff
@@ -0,0 +1,13 @@
+// Session.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Синглтон для хранения данных о текущей сессии
+final class Session {
+    static let shared = Session()
+    private init() {}
```

Comment:
```suggestion
MARK: Init 
    private init() {}
```
и опусти его ниже public properties